Google.PowerShell/Sql/GcSqlInstanceCmdlets.cs

```diff
@@ -149,20 +164,94 @@ public class AddGcSqlInstanceCmdlet : GcSqlCmdlet
         /// Can be created with New-GcSqlInstanceConfig.
         /// </para>
         /// </summary>
-        [Parameter(Mandatory = true, Position = 0, ValueFromPipeline = true)]
+        [Parameter(Mandatory = true, Position = 0, ValueFromPipeline = true,
+            ParameterSetName = ParameterSetNames.ByConfig)]
         public DatabaseInstance InstanceConfig { get; set; }
 
+        /// <summary>
+        /// <para type="description">
+        /// The instance resource. 
+        /// Can be created with New-GcSqlInstanceConfig.
+        /// </para>
+        /// </summary>
+        [Parameter(Mandatory = true, Position = 0, 
+            ParameterSetName = ParameterSetNames.Default)]
+        public string Name { get; set; }
+
+
+        /// <summary>
+        /// Creates a default Google Cloud SQL Database instance.
+        /// </summary>
+        /// <returns> A SQL instance object. </returns>
+        private DatabaseInstance createDefaultInstance()
+        {
+            DatabaseInstance instance = new DatabaseInstance
+            {
+                Settings = new Settings {
+                    Tier = "db-n1-standard-1",
+                    PricingPlan = "PER_USE",
+                    ActivationPolicy = "ALWAYS",
+                    BackupConfiguration = new BackupConfiguration
+                    {
+                        BinaryLogEnabled = true,
+                        Enabled = true,
+                        Kind = "sql#backupConfiguration",
```

Comment:
You shouldn't need to set this, right? Can we get away without setting this string?